server/liveblog/embed/embed_assets/default-theme/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "default-theme",
+    "version": "1.0",
+    "angularModule": "liveblog.default-theme",
+    "styles": [
+        "styles/embed.css",
+        "//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css"
+    ],
+    "scripts": [
+        "//code.angularjs.org/1.3.14/angular-sanitize.js",
+        "//code.angularjs.org/1.3.14/angular-animate.js",
+        "main.js"
```

Comment:
why not then add also angular?
Main the outside of the theme, so it shouldn't be in the theme.